neocore/Cryptography/Helper.py

```diff
@@ -140,11 +140,11 @@ def base256_encode(n, minwidth=0):  # int/long to byte array
         n (int): input value.
         minwidth: minimum return value length.
 
-    Raises:
-        ValueError: if a negative number is provided.
-
     Returns:
         bytearray:
+
+    Raises:
+        ValueError: if a negative number is provided.
```

Comment:
```suggestion
        ValueError: if a negative number is provided for `n`.
```
there are 2 arguments that could cause this, but it's only for the mandatory argument `n`